fn_apility/fn_apility/lib/resilient_common.py

```diff
@@ -0,0 +1,21 @@
+# (c) Copyright IBM Corp. 2010, 2018. All Rights Reserved.
+
+
+def api_url(query_type):
+    """
+    Method Will return appropriate URL based on query type    
+    """
+    type_url_mapping = {'Email': 'bademail', 'Domain': 'baddomain', 'IP Address': 'v2.0/ip'}
```

Comment:
Should you also include 'Email Sender', 'Email Sender Name' and 'Email Recipient' ?

Reply:
I believe these 3 we discussed for this release/version.

Reply:
Let's include Email Sender and Email Recipient as those will specifically addresses like user@company.co